public/index.html

```diff
@@ -2,7 +2,13 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title><%= foo %></title>
+    <title><%= title %></title>
+    <!-- load source styles here -->
+    <link href="../src/style.css" rel="stylesheet">
+    <!-- load source scripts here -->
+    <script src="../src/index.js" defer="defer"></script>
```

Comment:
Do I have to manually adding these resource links into the template file? Could it be automatically inserted like what `html-webpack-plugin` does?

Reply:
The goal of this plugin - avoid "black magic" (injecting somewhere in HTML) and to have full controll over all styles and scripts in generated HTML.

So and so you must define somewhere source styles and scripts.
Instead of to define JS in Webpack Entry and importing SCSS in JS, source files can be specified at right place - directly in HTML.

The profit - you specifiy all source styles and scripts at one correct place, instead of definitions in many places (JS in Webpack Enty, import SCSS in JS).

Reply:
> The profit - you specifiy all source styles and scripts at one correct place, instead of definitions in many places (JS in Webpack Enty, import SCSS in JS).

That makes great sense.